src/main/resources/feedbackQuestionConstSumEditFormTemplate.html

```diff
@@ -44,6 +49,7 @@
                     <div class="col-xs-6 padding-0">
                         <label class="margin-top-7px padding-left-7px">in total</label>
                     </div>
```

Comment:
Looks okay if you expand the unmodified parts, there's a closing `</div>` in there